.gitlab-ci.yml

```diff
@@ -46,6 +46,11 @@ build-images:
   artifacts:
     reports:
       dotenv: build.env
+  after_script:
+    - >
+      if [ $CI_JOB_STATUS != 'success' ]; then
+        curl -X 'PUT' "${CNS_TESTBEDPOOL_SVC_RELEASE_URL}/${CNS_CSI_ID}" -H 'accept: application/json' -u "${CNS_MANAGER_USERNAME}:${CNS_MANAGER_PASSWORD}"
```

Comment:
Why do we need to release the testbed in the `build-images` job?

Reply:
My latest changes were not reflected at the time of review - sorry for the inconvenience

New changes are committed now.